inc/emergency-banner.php

```diff
@@ -2,61 +2,87 @@
 
 if( function_exists('get_field') ) {
 
+    $banner_type = get_field('banner_type', 'option');
 
-$active = get_field('banner_active', 'option');
+    $banner_type_class = "emergency-banner";
 
-if($active) {
-    ?>
-    <div
-        class="emergency-banner"
-        style="
-		background-color: #0b0c0c;
-		color: #fff;
-		padding: 2rem 3rem 3rem;
-		border-bottom: 2px solid #f1f2f3;
-	">
-        <div style="
-			max-width:960px;
-			margin:0 auto;"
-        >
-            <div style="
-				font-weight: bold;
-				font-size: 48px;
-				padding-bottom: 2rem;
-			">
-                <?php
-                echo get_field('banner_title', 'option');
-                ?>
-            </div>
-            <div style="
-				padding-bottom: 2rem;
-			">
+    $banner_title = $banner_subtext = "";
+
+    switch ($banner_type) {
+        case "No banner":
+            $active = false;
+            break;
+        case "National emergency":
+            $active = true;
+            $banner_type_class .= "--national-emergency";
+            $banner_title = get_field('banner_national_emergency_title', 'option');
+            $banner_subtext = get_field('banner_summary', 'option');
+            if (is_null($banner_title) || trim($banner_title) == "" ) {
+                if (is_null($banner_subtext) || trim($banner_subtext) == "" ) {
```

Comment:
You have made the summary required here and yet the field isnt..should this be required?

Reply:
This is a "nothing entered" check.  If both title and summary are blank, then the banner is blank.  This should not happen, but might if someone enters a space for each field.

The title can revert to the default title if only that is blank, but if both are blank, we remove the banner.  

I have added a safeguard against the default being left in the title but no summary, which is more likely than the above.